### ISSUE TITLE
Spawn more than one agent at the same time.

### ISSUE THREAD

**Issue by gallandarakhneorg** [sarl/sarl#481]
It will be nice to spawn more than one agent of a given type at the same time.
The BICs must be updated accordingly.

```
def spawn(type : Class<? extends agent, nb : int = 1, args : Object*) : UUID
```

I think also that the platform could emit only one AgentSpawned instance per call to the spawning function. In other words, one occurrence of the the agent may be for a group of agents. I think it will save running time.

**Comment by gallandarakhneorg** [comment]
We need to discuss on the prototype of the function. Indeed, the prototype proposed above could cause issues with old code.
Indeed, `spawn(typeof(AgentType), 2, 3)` (two arguments to give to the agent) with the current API will not have the same outcomes with the proposed prototype.

**Comment by gallandarakhneorg** [comment]
This issue should be fixed with Issue #76.
